### ISSUE TITLE
Print summary every second

### ISSUE THREAD

**Issue by ppnaik1890** [rakyll/hey#231]
Can I make hey print report every second?  I did not find such a command line argument. It would be really helpful if you could guide me on this. Thanks!

**Comment by bmwant** [comment]
Yes, it would be great to have at least some progress bar showing the state of things

**Comment by mark4z** [comment]
see my fork, https://github.com/mark4z/hey
